components/todo-input.js

```diff
@@ -1,20 +1,96 @@
-import React from "react";
-import styled from "styled-components";
+import React from 'react';
+import styled from 'styled-components';
 
-export default function TodoInput() {
-  return (
-    <Wrapper>
-      리액트-투두
-      <br />
-      시간을 입력하는 칸
-      <br />
-      Todo를 입력하는 칸
-    </Wrapper>
-  );
+export default function TodoInput(props) {
+	const { date, todo, onChangeTodo, onChangeDate, onSubmit } = props;
+
+	const maxLengthCheck = (object) => {
+		if (object.value.length > object.maxLength) {
+			object.value = object.value.slice(0, object.maxLength);
+		}
+	};
+
+	return (
+		<Form>
+			<InputSection>
+				<P>시간</P>
+				<Input
+					maxlength="8"
+					oninput="maxLengthCheck(this)"
+					type="number"
+					placeholder="날짜를 입력하세요 (ex.20200404)"
+					value={date}
+					onChange={onChangeDate}
+				/>
+			</InputSection>
+
+			<InputSection>
+				<P>TODO</P>
+				<TextArea
+					placeholder="할 일을 입력하세요"
+					value={todo}
+					onChange={onChangeTodo}
+				/>
+			</InputSection>
+			<Button onClick={onSubmit}>등록</Button>
+		</Form>
+	);
 }
 
-const Wrapper = styled.div`
-  border: solid 1px;
-  font-size: 18px;
-  flex: 1;
+const Input = styled.input`
+	width: 80%;
+	border-width: 1px;
+	border-style: solid;
+	border-color: rgb(97, 97, 97);
+	border-image: initial;
+	padding: 0.5rem 0.8rem;
+`;
+const TextArea = styled.textarea`
+	width: 80%;
+	height: 30vh;
+	resize: none;
+	border-width: 1px;
+	border-style: solid;
+	border-color: rgb(97, 97, 97);
+	padding: 0.5rem 0.8rem;
+`;
+const P = styled.p`
+	font-size: 1.5rem;
+	color: white;
+	margin: 0;
+`;
+const InputSection = styled.div`
+	color: white;
+	width: 100%;
+	display: flex;
+	flex-direction: row;
+	-webkit-box-pack: justify;
+	justify-content: space-between;
+	align-items: flex-start;
+	margin-bottom: 3rem;
+`;
```

Comment:
```suggestion
const Row = styled.div`
	color: white;
	width: 100%;
	display: flex;
	flex-direction: row;
	justify-content: space-between;
	margin-bottom: 3rem;
`;
```
1. ```InputSection```이라는 네이밍은 자칫 잘못하면 form을 감싸는 태그로 오해할 가능성이 있어 보여요.
Input label과 Input을 한 행에 오도록 정렬하는 태그라는 의미에서 ```Row```라는 네이밍이 좋을 것 같아요 :)
2. ```box-pack```은 표준 문법이 아닙니다. 지우셔도 똑같이 작동할 거예요 [참고](https://developer.mozilla.org/en-US/docs/Web/CSS/box-pack)
3. ```align-items: flex-start```는 없어도 스타일 똑같을 거예요!